src/Components/AdvancedSearch/AdvancedSearchModal.tsx

```diff
@@ -89,9 +112,11 @@ const AdvancedSearchModal: React.FC<IAdvancedSearchProps> = (props) => {
                     </div>
                     <div className={classNames.resultsContainer}>
                         <AdvancedSearchResultDetailsList
-                            twins={filteredTwins}
-                            searchedProperties={cols}
-                            adapter={new MockAdapter()}
+                            twins={filteredTwins.current}
+                            searchedProperties={Array.from(
```

Comment:
can we make this const var with useMemo insread of creating the array each time?

Reply:
I'm not sure the creation process is going to be very intensive. Based on [the guidance](https://overreacted.io/before-you-memo/) I've read it seems they recommend you avoid memo unless it's truly an expensive operation.